content/warnings/refs-must-have-owner.md

```diff
@@ -4,45 +4,47 @@ layout: single
 permalink: warnings/refs-must-have-owner.html
 ---
 
-You are probably here because you got one of the following error messages:
+Jesteś tu najprawdopodobniej z powodu jednego z dwóch błędów:
 
 *React 16.0.0+*
 > Warning:
 >
 > Element ref was specified as a string (myRefName) but no owner was set. You may have multiple copies of React loaded. (details: https://fb.me/react-refs-must-have-owner).
 
-*earlier versions of React*
+*wcześniejsze wersje Reacta*
 > Warning:
 >
 > addComponentAsRefTo(...): Only a ReactOwner can have refs. You might be adding a ref to a component that was not created inside a component's `render` method, or you have multiple copies of React loaded.
 
-This usually means one of three things:
+Zazwyczaj oznacza to jedną z trzech rzeczy:
 
-- You are trying to add a `ref` to a function component.
-- You are trying to add a `ref` to an element that is being created outside of a component's render() function.
-- You have multiple (conflicting) copies of React loaded (eg. due to a misconfigured npm dependency)
+- Usiłujesz dodać referencję (ang. `ref`) do komponentu funkcyjnego.
```

Comment:
```suggestion
- Usiłujesz dodać referencję (`ref`) do komponentu funkcyjnego.
```
To nie jest bezpośrednie tłumaczenie, więc nie piszemy języka. Raczej wartość używaną w kodzie.